performance/benchmarks/bm_telco_fractions.py

```diff
@@ -0,0 +1,100 @@
+#-*- coding: UTF-8 -*-
+
+""" Telco Benchmark for measuring the performance of Fraction calculations
+
+http://www2.hursley.ibm.com/decimal/telco.html
+http://www2.hursley.ibm.com/decimal/telcoSpec.html
```

Comment:
HTTP 404 error :-/

Hum, in a different flavor of the bm_telco.py benchmark, I found this link. Is it accurate?
http://speleotrove.com/decimal/telco.html